### DISCUSSION TITLE
Horizon local installation design

### DISCUSSION THREAD

**Discussion by cubxxw** [horizoncd/horizon#81] [Ideas]
https://nsddd.notion.site/Horizon-light-install-541273230ed646948b58f904aa56ec39

**Comment by cubxxw** [comment]
+ [ ] [https://github.com/k3s-io/k3s](https://github.com/k3s-io/k3s)

**Comment by kilosonc** [comment]
The first solution not only consumes fewer resources, but also makes it more convenient for local debugging and development by using the independently deployed k3s solution, compared to using kind to launch a k8s cluster inside a container.

The second solution is to pull all the images instead of just the Horizon image, and when using kind to start, mount all the images to the kind image directory, so that k8s does not need to download the images.
